### ISSUE TITLE
Any possibility to deactivate specific input types

### ISSUE THREAD

**Issue by grasdaggel** [tzyganu/Magento2SampleModule#15]
We want to use your module for testing purposes on a Mage 2.1.1 env. 

I'm struggeling right now to deactivate specific Input fields, like DOB, Awards and Type (generic stuff). 

Deactivating the virtual-Nodes in the `etc/di.xml` doesn't do the job, theres only an exception. Is there a simple way to deactivate specific input types, or do I need to modify the whole module in order do sort out different input fields? 

Thanks to whom it may concern, your help is very appreciated.

**Comment by tzyganu** [comment]
@grasdaggel You can take the extension and remove any fields you want from the [Setup/Install.php](https://github.com/tzyganu/Magento2SampleModule/blob/master/Setup/InstallSchema.php#L78) and from the ui components:  [list](https://github.com/tzyganu/Magento2SampleModule/blob/master/view/adminhtml/ui_component/sample_news_author_listing.xml#L300) and [form](https://github.com/tzyganu/Magento2SampleModule/blob/master/view/adminhtml/ui_component/sample_news_author_form.xml#L144).  
Then you will have to edit the interfaces and the frontend templates to delete what you need.  
Basically search for any occurrence of `dob`  and remove what you don't want.  
Only after you are done removing everything, install the extension.
